tests/foreman/cli/test_remoteexecution.py

```diff
@@ -134,6 +134,40 @@ def test_positive_run_default_job_template(
         assert 'Exit' in out
         assert 'Internal Server Error' not in out
 
+    @pytest.mark.tier3
+    @pytest.mark.rhel_ver_list([8])
+    def test_positive_timeout_to_kill(self, module_org, rex_contenthost, module_target_sat):
+        """Use timeout to kill setting to cancel the job
+
+        :id: 580f886b-ac24-4afa-9aca-e9afc5cfdc9c
+
+        :expectedresults: Verify the job was killed after specified times
+
+        :parametrized: yes
+        """
+        client = rex_contenthost
+        command = 'sleep 160'
+        invocation_command = module_target_sat.cli_factory.job_invocation(
+            {
+                'job-template': 'Run Command - Script Default',
+                'inputs': f'command={command}',
+                'search-query': f"name ~ {client.hostname}",
+                'timeout_to_kill': '5',
```

Comment:
Per PRT failure I guess this could help
```suggestion
                'timeout-to-kill': '5',
```